pkg/helper/maps.go

```diff
@@ -0,0 +1,42 @@
+package helper
+
+import (
+	"reflect"
+)
+
+var (
+	MaxDepth = 16
+)
+
+func Merge(dst, src map[string]interface{}) map[string]interface{} {
```

Comment:
We may add a todo here once we move to go 1.18
There is a new maps copy function that would do the job:
https://pkg.go.dev/golang.org/x/exp/maps#Copy

Reply:
Being picky, you don't need to return `dst` as maps are treated as references.